crates/consensus/src/receipt/any.rs

```diff
@@ -39,31 +39,28 @@ impl AnyReceiptEnvelope {
             length + 1
         }
     }
+}
 
-    /// Return true if the transaction was successful.
-    pub const fn is_success(&self) -> bool {
-        self.status()
-    }
-
+impl TxReceipt for AnyReceiptEnvelope {
```

Comment:
this should now be generic over `T` 

`impl<T> TxReceipt for AnyReceiptEnvelope<T>`